proto/devices_state.proto

```diff
@@ -31,6 +31,8 @@ message DeviceBehavior {
     PortBehavior.Behavior port_behavior = 3;
     PortBehavior.AutoSequestering auto_sequestering = 4;
   }
+  // ISO8601 timestamp string for manual sequestering
+  string manual_sequestering_timestamp = 6;
```

Comment:
this isn't necessarily going to be "manual" so I'd leave that qualifier out. Mechanically, what does this mean? Maybe next_sequestering_timestamp?

Reply:
I want to differentiate the auto and manual. This sequestering timestamp may not be the actual next sequestering timestamp since between the time the config is read and the timestamp, device can go offline and online again which will trigger the auto sequestering. I think scoping this timestamp to manual is a better option.